config/routes/api.rb

```diff
@@ -81,6 +81,7 @@
           patch :start
         end
 
+        resources :export_solutions, only: [:index], controller: "export_solutions"
```

Comment:
@iHiD Is this the way you'd like the controller to be named?

Reply:
Yes, which means you can leave the `controller` param out altogether on this line.